packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/README.md

```diff
@@ -9,6 +9,17 @@ The Bar Chart Viewer...
 - allows users to view bar chart data (x-axis: categorical values, y-axis:
   numerical values for each category)
 
+## Props
+
+- `barStyles` _(object)_ An object containing properties for the individual bar styles. `padding` is the available configurable style and sets the space between each bar. It defaults to `0.25`.
+- `data` _(array)_ An array of objects. Each object must contain a string label and a number value. A string color value is optional for each bar and can be set to either a string variable name from the [Zooniverse theme](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-grommet-theme) or any hex value. A color variable name is preferred from the theme since it has been approved by the Zooniverse designer and tested for accessibility. This should be set by the data property in the location JSON of the subject. See the section on [JSON file](#JSON_file) for more information. Required.
```

Comment:
Is this an array of `{ label, value }` objects? The description isn't clear and I don't follow how the color value is added. When you say 'each bar' do you mean each item in the array? What name does the color value take in the data object?

Maybe we should enable the source code viewer in the storybook, so that people can see how the examples are set up?

Reply:
The link to the zooniverse theme has a readme where I've added a table of the available colors and their variable names. There's enough of them that I don't want to include the full list here, so I just link to it instead.

The JSON file section has an example of the shape of the data, which is also linked to. 

We could add that storybook plugin in a separate PR?